code/modules/research/designs/misc_designs.dm

```diff
@@ -67,6 +67,17 @@
 //////////////////Misc///////////////////
 /////////////////////////////////////////
 
+
+/datum/design/plasticducky
+	name = "Rubber Ducky"
+	desc = "The classic Nanotracen design for competitively priced bath based duck toys. No need for fancy Waffle co. rubber, buy Plastic Ducks today!"

```

Comment:
please fix the spelling

Reply:
your branch also now conflicts with the existing designs because you "copy pasted them because I was lazy" which is why you should not be lazy and do it right the first time.

Reply:
Thank you for your feedback.